### DISCUSSION TITLE
Olá, pessoal.  Estou começando agora com Budibase e tenho interesse criar aplicações externas, self-hosted, porém a documentação recomenda o Budibase para aplicações para times internos. Gostaria de saber se existe alguma tipo de limitação ou restrição para desenvolver aplicações externas ou até mesmo se não é recomendado.   Muito obrigado.

### DISCUSSION THREAD

**Discussion by thiagoalgo** [Budibase/budibase#12317] [General]
Olá, pessoal.

Estou começando agora com Budibase e tenho interesse criar aplicações externas, self-hosted, porém a documentação recomenda o Budibase para aplicações para times internos. Gostaria de saber se existe alguma tipo de limitação ou restrição para desenvolver aplicações externas ou até mesmo se não é recomendado. 

Muito obrigado.

**Comment by shogunpurple** [comment]
Olá @thiagoalgo

Realmente não há uma limitação para desenvolver aplicações externas, mas o Budibase não foi construído para esse tipo de uso. As aplicações criadas pelo Budibase precisam de um login, a menos que você torne a ecrã pública e exponha seus dados abertamente. 

Se você quer construir uma aplicação para inscrição – onde os usuários podem preencher um formulário para se inscreverem – será mais difícil.

Esta discussão pode ser útil - https://github.com/Budibase/budibase/discussions/6098 um usuário na nossa comunidade criou um template para desenvolver aplicações externas. No entanto, não é uma configuração que suportamos oficialmente. O que você quer construir?

Se possível, peço por favor que use inglês nas discussões para que seja mais fácil para a equipa responder. Obrigado!

**Reply by thiagoalgo** [reply]
Thanks for the response, Martin.

Sorry for the confusion in the language. I'm not fluent in English, I used a translator and copied the wrong text. As it was early in the morning, I ended up not noticing the confusion. Please excuse me.

The application is to run in a closed environment where the customer can set up pre-defined automation flows. These automations will run outside the application on another infrastructure.

We can therefore consider that the system will be closed access to pre-registered customers who will be able to register their teams to carry out CRUD operations.

Based on your explanation, I believe that BudiBase can meet the requirements, right?

**Reply by shogunpurple** [reply]
No problem at all - that's ok!

This sounds like a very good use case for budibase. You could definitely build this with our platform.

Yep, perfect - if you will pre register the customers then that's absolutely fine and budibase will suit your needs well, budibase users can invite other users to the platform as well.
